plugins/woocommerce/tests/e2e-pw/tests/merchant/create-variable-product.spec.js

```diff
@@ -40,6 +40,32 @@ test.describe.serial( 'Add New Variable Product Page', () => {
 		await api.post( 'products/batch', { delete: ids } );
 	} );
 
+	test( 'shows the variable product tour', async ( { page } ) => {
```

Comment:
This is kind of tricky because my e2e tests are failing, but after running them a couple of times, I realized that it was because this only will pass successfully the first time we run tests.
How can we solve that problem?